website/blog/2020-12-11-plugin-ruby-1.0.md

```diff
@@ -0,0 +1,42 @@
+---
+author: Kevin Deisz (@kddeisz)
+authorURL: https://twitter.com/kddeisz
+title: Prettier for Ruby goes v1.0 🎉
+---
+
+After more than 1400 commits since July 2018, we're happy to announce that we're releasing v1.0 of [Prettier for Ruby](https://github.com/prettier/plugin-ruby). In this blog post, we'd like to give a short overview of how the plugin works, its philosophy, and what to expect in the future.
+
+<!--truncate-->
+
+## How does it work?
+
+Prettier for Ruby works through Prettier's [plugin API](https://prettier.io/docs/en/plugins.html). Its `parse` function works by spawning a Ruby process and using Ruby's own parser (known as `Ripper`). After tracking all of the various node types, comments, location information, and other various metadata, it returns a built abstract syntax tree (AST) to the Prettier process. Finally it converts that AST into Prettier's intermediate representation (known as `Doc`) before allowing Prettier to handle printing it back out.
```

Comment:
```suggestion
Prettier for Ruby works through Prettier's [plugin API](https://prettier.io/docs/en/plugins.html). It's `parse` function works by spawning a Ruby process and using Ruby's own parser (known as `Ripper`). After tracking all of the various node types, comments, location information, and other various metadata, it returns a built abstract syntax tree (AST) to the Prettier process. Finally it converts that AST into Prettier's intermediate representation (known as `Doc`) before allowing Prettier to handle printing it back out.
```

Reply:
Nah that's actually correct. `It's` would be a contraction of `It is parse function`.